packages/translate/src/utils.ts

```diff
@@ -125,11 +125,9 @@ export const lintLocaleFiles = async (
     targetPackages.map(async (packageName) => {
       const phrasesPath = path.join(instancePath, 'packages', packageName);
       const localesPath = path.join(phrasesPath, 'src/locales');
-      await execPromise(
-        'pnpm',
-        ['eslint', '--ext', '.ts', path.relative(phrasesPath, localesPath), '--fix'],
-        { cwd: phrasesPath }
-      );
+      await execPromise('pnpm', ['eslint', '--ext', '.ts', localesPath, '--fix'], {
```

Comment:
a breaking change?

Reply:
No, tested in Console and it was working fine

<img width="237" alt="image" src="https://github.com/user-attachments/assets/a1f75bbf-4056-4dd0-bad8-b81e2e3225c4" />